source/h1_connection.c

```diff
@@ -518,6 +522,73 @@ struct aws_http_stream *s_new_client_request_stream(const struct aws_http_reques
     return NULL;
 }
 
+struct h1_stream *new_server_stream(struct h1_connection *connection) {
```

Comment:
This function can be `static`, and we usually start static function names with `s_` (see other functions in this file as an example)